src/components/DeleteHelicopter.jsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  fetchHelicopters,
+  deleteHelicopter,
+  updateHelicopters,
+} from "../features/helicopters/helicopterSlice";
+import LayoutComponent from "../Layout";
+import { Button, Table, Popconfirm } from 'antd';
+import { QuestionCircleOutlined } from "@ant-design/icons";
+
+const DeleteHelicopterComponent = () => {
```

Comment:
- [ ] When I go to "delete reservation" I get an unrecoverable blank page:

![image](https://github.com/emmanuelsteven/heli-hire-captsone-frontend/assets/57421823/9859d882-169b-4e8c-a4ff-b17c7f5c9cac)

Kindly, make it so the page does not crash when I go to "delete reservation". 👍

Reply:
fixed